CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 ------------
 
+### Changed
```

Comment:
Should minor security fixes like the one in this PR also go into the changelog?

Reply:
I think the fix is not included yet, mainly because we didn't turn on the deep copy. As a result, using edl to generate data structures does not bring any changes. I need to check and manually insert checks either in this PR or creating another one.

Reply:
FYI: deepcopy is still an experimental feature per #1839 (requiring `--experimental` flag).

Reply:
@jhand2 I think this is appropriately under `Changed` since the existing bits headers are public headers and all public changes should be documented in the CHANGELOG.md.

Reply:
Oh good point. I would prefer checks in a different PR or changing deep copy to no longer be an experimental feature